manual/mapper/daos/delete/README.md

```diff
@@ -36,6 +36,28 @@ The method can operate on:
     In addition, because the entity class can't be inferred from the method signature, it must be
     specified via the annotation's `entityClass` element.
 
+* a subset of the primary key.  As in the partition key, or partition key + subset of clustering 
+  columns:
+
+    ```java
+    // given: PRIMARY KEY ((product_id, day), customer_id, ts)
+    // delete all rows in partition
+    @Delete(entityClass = ProductSale.class)
+    void deleteByIdForDay(UUID productId, LocalDate day);
+
+    // delete by partition key and partial clustering key
+    @Delete(entityClass = ProductSale.class)
+    void deleteByIdForCustomer(UUID productId, LocalDate day, UUID customerId);
+   
+    /* Note that the clustering columns in your primary key definition are significant. All
+     * proceeding clustering columns must be provided if any are.
```

Comment:
> proceeding clustering columns

Did you mean "preceding"?

Reply:
:man_facepalming: yep, that's what I meant.  Good catch, i'll fix.